### PR TITLE
Restarts reformat

### PR DESCRIPTION
**Pull request by robert-bryson**
Related to https://github.com/GSA/data.gov/issues/3326

### CODE REVIEW

**Comment by robert-bryson** [comment]
This [should be working](https://github.com/GSA/inventory-app/actions/runs/3989134618) for restarts. I would like to merge this template and then make the related PRs for [inventory](https://github.com/GSA/inventory-app/pull/536) and catalog. Once those are in and working, I'll make the PRs that would effect deploys (the other half of this story).

File: bin/check-running-job [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    # If there is more than one instance of 'instances', the app is already restarting
+    if [[ $(cf app "$app_to_check" | grep -c '^instances:') -gt 1 ]]; then
+      echo "Deployment or Restart in progress... not doing anything"
+      exit 0
+    fi
+}
+
+# attempting https://stackoverflow.com/a/29967433/20735210
+if [[ "${BASH_SOURCE[0]}" == "${0}" ]]; then
```
Comment (nickumia-reisys) [inline bin/check-running-job:31]: Is this only because you are `source`ing this file?

Comment (robert-bryson) [reply bin/check-running-job:31]: It's to mimic a python-like `if __name__ is __main__` type thing so I can use it directly as well as in the `restart-app` script.

Comment (nickumia-reisys) [reply bin/check-running-job:31]: Fair enough, so only for local testing?

**Review by nickumia-reisys — APPROVED** [review]
🤖

**Review by jbrown-xentity — APPROVED** [review]

**Pull request merged**